utils/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ if (BUILD_D)
   if(CMAKE_C_COMPILER_ID MATCHES "^(Intel|IntelLLVM)$")
     set_target_properties(copygb2 PROPERTIES COMPILE_FLAGS "-r8 -auto -convert big_endian -fpp")
   elseif(CMAKE_C_COMPILER_ID MATCHES "^(GNU|Clang|AppleClang)$")
-    set_target_properties(copygb2 PROPERTIES COMPILE_FLAGS "-fdefault-real-8 -fconvert=big-endian -cpp")
+    set_target_properties(copygb2 PROPERTIES COMPILE_FLAGS "-fdefault-real-8 -fconvert=big-endian -cpp -frecursive")
```

Comment:
It's not good to rely on compiler flags. What warnings does this new flag address?

Reply:
Here is the warning:

![Screenshot 2024-07-01 073041](https://github.com/NOAA-EMC/NCEPLIBS-g2/assets/166434581/7f8dee87-f2da-4d20-9861-c41786c31c00)

I could also increase the limit like the warning suggests?